spec/text_to_braille_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'simplecov'
+SimpleCov.start
+require './lib/file_translate'
+require './lib/text_to_braille'
+require 'rspec'
+require 'pry'
+
```

Comment:
created spec file for each method in text_to_braille.rb, other than the start method which just returns nil due to the puts statement.